### PR TITLE
Upgrade to dotnet 3

### PR DESCRIPTION
**Pull request by askingalot**
Resolves
https://github.com/nashville-software-school/bangazon-inc/issues/194

### CODE REVIEW

**Comment by askingalot** [comment]
I tried my damnedest to use the .net core 3.0 json deserializer instead of newtonsoft in the tests, but I couldn't manage to globally set the JsonNamingPolicy set to camel case. It works if you pass options to each Deserialize call, but that's just not worth it...

File: TestBangazonAPI/APIClientProvider.cs [hunk 1]
```diff
@@ -2,25 +2,14 @@
...
-                builder.ConfigureAppConfiguration((webbuilder, configbuilder) =>
-                {
-                    configbuilder
-                        .SetBasePath(webbuilder.HostingEnvironment.ContentRootPath)
-                        .AddJsonFile("appsettings.Testing.json")
-                        .AddEnvironmentVariables();
-                    configbuilder.Build();
-                });
-            });
+        private readonly WebApplicationFactory<Startup> _factory =
```
Comment (stevebrownlee) [inline TestBangazonAPI/APIClientProvider.cs]: Curious about what your plan is for using a test database for the sprint. I noticed you took out at the testing app settings.

File: BangazonAPI/Startup.cs [hunk 2]
```diff
@@ -1,35 +1,20 @@
...
 {
     public class Startup
     {
-        public Startup(IConfiguration configuration)
-        {
-            Configuration = configuration;
-        }
-
-        public IConfiguration Configuration { get; }
-
```
Comment (stevebrownlee) [inline BangazonAPI/Startup.cs]: Are these not needed in 3.0? That's cool

**Comment by stevebrownlee** [comment]
:shipit: Thanks, Andy

**Pull request merged**